capstone/src/main/resources/static/event.js

```diff
@@ -1,9 +1,9 @@
 async function getEvents() {
   const response = await fetch('get-all-events');
   const jsonEvents = await response.json();
-  const eventList = document.getElementById('events-container');
+  //const eventList = document.getElementById('events-container');
```

Comment:
This page won't be showing events anymore? I guess it makes sense, since they are already searchable from the index page. Consider redirecting there from the event save then.

Reply:
I will create an issue for it!